content/klc/F5.1.adoc

```diff
@@ -19,7 +19,8 @@ The following elements must be provided on the silkscreen.
   * `0.1mm` is allowed for high density designs
   * `0.15mm` is allowed for low density designs
 . Silkscreen must not be placed over pads or areas of exposed copper
-  * Clearance between silkscreen and exposed copper elements must be at least the silkscreen line width or more than the pad mask expansion, whichever is greater.
+  * Clearance between silkscreen and exposed copper elements is recommended to be 0.2mm.
+  * Clearance must be at least the silkscreen line width or more than the pad mask expansion, whichever is greater.
```

Comment:
Is it intentional that it now says >= line width and > expansion? If not, this could perhaps be simplified to "Clearance must be at least the silkscreen line width or the pad mask expansion, whichever is greater".

Reply:
Updated text as suggested.